### PR TITLE
Group Join updates

### PR DESCRIPTION
**Pull request by OSPFNeighbour**
Fixed up the screen theme to match the rest of the app.

Setup the query so that searching for a group searches for a group against the server via a query param.

Fixed some component margins

### CODE REVIEW

**Comment by OSPFNeighbour** [comment]
screenshot 2018-03-13 19 13 24
screenshot 2018-03-13 19 13 13
screenshot 2018-03-13 19 13 03

File: client/src/screens/events/Detail.js [hunk 1]
```diff
@@ -268,9 +268,9 @@ class EventDetail extends Component {
 
     if (!event) {
       return (
-        <View style={styles.container}>
+        <Container>
```
Comment (sjmcculloch) [inline client/src/screens/events/Detail.js]: Nice

File: client/src/screens/events/EventUsers.js [hunk 2]
```diff
@@ -59,7 +59,7 @@ class EventUsers extends Component {
             break;
           case 'attending':
             tempResponse.etaText = tempResponse.eta === 0 ? '' : `- ETA ${moment.unix(tempResponse.eta).fromNow()}`;
-            tempResponse.statusText = tempResponse.detail === ''
+            tempResponse.statusText = !tempResponse.detail || tempResponse.detail === ''
```
Comment (sjmcculloch) [inline client/src/screens/events/EventUsers.js]: Wasn't this in the other PR?

Comment (OSPFNeighbour) [reply client/src/screens/events/EventUsers.js]: weird. must have sucked in an old commit

**Pull request merged**